heat_pump_adoption_modelling/analysis/MCS/plot_mcs.py

```diff
@@ -0,0 +1,549 @@
+# File: heat_pump_adoption_modelling/analysis/MCS/plot_mcs.py
+"""Produce plots relating to heat pump costs/capacities:
+- Counts by manufacturer
+- Median costs by year and tech type
+- Mean ASHP SCOP by year and flow temp
+- Costs by ASHP capacity
+- ASHP capacities by property archetype
+- ASHP costs by property archetype
+
+Current draft of paper can be found here: https://docs.google.com/document/d/1Es-eWt5kLTbsZRtazi23IbpaPIdPDe29juqWGnAGKHc/edit?usp=sharing
+"""
+
+import pandas as pd
+import numpy as np
+import matplotlib.pyplot as plt
+
+from heat_pump_adoption_modelling import PROJECT_DIR
+from heat_pump_adoption_modelling.getters.load_mcs import load_inflation
+from heat_pump_adoption_modelling.pipeline.preprocessing.mcs_epc_joining import (
+    join_mcs_epc_data,
+)
+
+# Load data for plotting
+
+
+def plottable_data(file_path=PROJECT_DIR / "outputs/mcs_epc.csv"):
+    """Load merged MCS/EPC data in a suitable form for plotting
+    (with additional variables: "built_post_1950" indicating
+    whether or not the property was built after 1950, and
+    "inflated_cost" indicating inflation-adjusted installation cost).
+
+    Parameters
+    ----------
+    file_path : str
+        Path to saved merged data.
+        If none, data is loaded from scratch.
+
+    Return
+    ----------
+    merged : pd.DataFrame
+        Merged MCS-EPC data with added variables.
+    """
+
+    if file_path is None:
+        merged = join_mcs_epc_data()
+    else:
+        merged = pd.read_csv(file_path)
+
+    # Add inflated cost variable
+    inflation = load_inflation()
+    merged = merged.merge(inflation, how="left", on="year")
+    merged["inflated_cost"] = merged["cost"] * merged["multiplier"]
+
+    # Add indicator variable for properties built after 1950
+    ages_dict = {
+        "England and Wales: before 1900": False,
```

Comment:
When I ran it with my EPC version, I got a KeyErrow for "Scotland: before 1919". Better add it to avoid future problems.

```suggestion
        "England and Wales: before 1900": False,
        "Scotland: before 1919" : False,
```

Reply:
Added, thanks!